recipes/cellsnake/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+ls -l
```

Comment:
please use `  script: {{ PYTHON }} -m pip install . -vv` in the build section of the yaml rather than a separate script (see other recent Python recipes for examples).